src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
```

Comment:
Replace the hard-coded value `7` with a constant named `LONG_TERM_RENTAL_DAYS` or similar, and replace `50` with a constant like `LONG_TERM_DISCOUNT`. This will make your code more readable and maintainable.